src/config.test.ts

```diff
@@ -148,4 +148,94 @@ describe("config", () => {
     const { config } = await import("./config.js");
     expect(config.options.scraping.daysBack).toBe(15);
   });
+
+  it("should support internal URLs in Actual Budget serverUrl", async () => {
+    const configWithInternalUrl = {
+      accounts: [{ companyId: "test", password: "pass", userCode: "12345" }],
+      storage: {
+        actual: {
+          serverUrl: "http://actual-budget:5006",
+          password: "test-password",
+          budgetId: "test-budget-id",
+          accounts: {},
+        },
+      },
+      options: {
+        scraping: {
+          daysBack: 15,
+          futureMonths: 1,
+          transactionHashType: "",
+          additionalTransactionInfo: false,
+          hiddenDeprecations: [],
+          maxParallelScrapers: 1,
+          domainTracking: false,
+        },
+        security: {
+          blockByDefault: false,
+        },
+        notifications: {},
+        logging: {
+          getIpInfoUrl: "https://ipinfo.io/json",
+        },
+      },
+    };
+
+    process.env = {
+      ...originalEnv,
+      MONEYMAN_CONFIG: JSON.stringify(configWithInternalUrl),
+    };
+
+    const { config } = await import("./config.js");
+    expect(config.storage.actual?.serverUrl).toBe("http://actual-budget:5006");
+  });
+
+  it("should support various internal URL formats", async () => {
+    const testCases = [
```

Comment:
Why not `it.each`??

Reply:
When fixing, also remove the other test you added and make sure you add all urls to the each